### ISSUE TITLE
Show video loading without controls

### ISSUE THREAD

**Issue by minas90** [brianwernick/ExoMedia#276]
Is it possible to separate loading progressbar from controls?

**Comment by brianwernick** [comment]
What do you mean?  Please explain in more details

**Comment by minas90** [comment]
When I use default controls with  app:useDefaultControls="true" it show loading progress with a big progressbar in center. I need to show it, but without controls.

**Comment by brianwernick** [comment]
You will have to provide your own `ProgressView` if you don't want the controls.  The current implementation will show loading when the video uri is set, and hide the progress view when `onPrepared()` is called

**Comment by minas90** [comment]
Ok, thanks. I already did so. :)
Anyway I think it is better to separate loading ProgressView from controls, because loading got nothing to do with the video player controls.
In some scenarios I don't need default ProgressView, but I do need controls and vice versa.

Details screen of my app has "full player", so it shows controls, but I have a custom fancy loading indicator. I copied your code from library and modified it to hide ProgressView and in the feed ( RecyclerView ) I do need the default ProgressView, but I don't need controls as it's redundant and consumes all touch events even if it shouldn't.
